### PR TITLE
docs: sphinx: Add pixel buffer reordering example

### PR DESCRIPTION
**Pull request by rleigh-codelibre**
See https://trello.com/c/citrfg1i/2-doc-add-example-code-for-the-interleaving

Testing: Check staging tutorial page

### CODE REVIEW

File: docs/sphinx/tutorial.rst [hunk 1]
```diff
@@ -531,6 +531,21 @@ defined storage order using the helper function:
...
+order; the dimension extents must be of the same size for the buffers
+to be compatible.  The following example demonstrates conversion of
+planar data to contiguous:
+
+.. literalinclude:: examples/pixeldata.cpp
+   :language: cpp
+   :start-after: reorder-example-start
+   :end-before: reorder-example-end
+
+In place conversion is not yet supported.
```
Comment (mtbc) [inline docs/sphinx/tutorial.rst]: In-place

Comment (rleigh-codelibre) [inline docs/sphinx/tutorial.rst]: Fix pushed, thanks.

**Comment by dgault** [comment]
The explanation and example here looks good and I can see it being beneficial for developers.

The pixeldata.cpp is building and running successfully and the latest docs build shows the tutorial page displaying correctly with the new example in place. The text reads fine to me also.

**Comment by sbesson** [comment]
See https://www.openmicroscopy.org/site/support/ome-files-cpp-staging/ome-files/manual/html/tutorial.html#pixel-data

**Pull request merged**